spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-schemas.spec.js

```diff
@@ -110,6 +110,74 @@ describe('Google Analytics schemas', function () {
     expect(returned).toEqual(expected)
   })
 
+  it('does not overwrite the schema is an undefined value is passed', function () {
```

Comment:
Possible typo: should `does not overwrite the schema is an undefined value is passed` be `does not overwrite the schema if an undefined value is passed`

Reply:
Fixed, thanks.